### PR TITLE
Fix my english

### PR DESCRIPTION
**Pull request by almet**
r? @n1k0 @Natim

### CODE REVIEW

**Comment by Natim** [comment]
Shall we create a redirection from the previous url?

**Comment by almet** [comment]
I don't think that's necessary (and I don't know how to do it, anyway).

**Comment by Natim** [comment]
I was planning to create an html file in the gh-pages with a `window.location.href`

**Comment by almet** [comment]
If you create it in the gh-pages it will be rewritten each time it's generated again. You would need to do so with pelican (it's possible, but more complex).

**Comment by Natim** [comment]
I was thinking of something like that: https://github.com/mlissner/michaeljaylissner.com/blob/master/content/blog/2010-donations/index.html

**Comment by almet** [comment]
see https://github.com/Nitron/pelican-alias

**Comment by Natim** [comment]
Oh yeah :+1:

**Comment by almet** [comment]
I wonder if that plays well with the l10n plugin though. Take care.

**Comment by almet** [comment]
Any news on this front?

**Comment by Natim** [comment]
I tried a bit but it didn't really work out.

**Comment by Natim** [comment]
Ok here we go.

File: Makefile [hunk 1]
```diff
@@ -32,6 +32,7 @@ $(PYTHON):
 
 html: install
 	$(PELICAN) $(INPUTDIR) -o $(OUTPUTDIR) -s $(CONFFILE) $(PELICANOPTS)
+	cp self-hosting-a-hard-promess.html $(OUTPUTDIR)/en/
```
Comment (almet) [inline Makefile:35]: oh, wow, that's a hackety hack (not very scalable)! That solves the current problem though, thanks!

**Pull request merged**